src/Abp/Application/Navigation/MenuItemDefinition.cs

```diff
@@ -37,10 +38,10 @@ public class MenuItemDefinition : IHasMenuItemDefinitions
         public string Url { get; set; }
 
         /// <summary>
-        /// A permission name. Only users that has this permission can see this menu item.
+        /// A permission dependency. Only users that can statify this permission dependency can see this menu item.
         /// Optional.
         /// </summary>
-        public string RequiredPermissionName { get; set; }
```

Comment:
I see. Is it fine if I create a ``SimplePermissionDependency`` for the ```requiredPermissionName``` and checking both ```SimplePermissionDependency(menuItemDefinition.RequiredPermissionName)``` and ```menuItemDefinition.PermissionDependency``` with ```PermissionDependencyContext```